cli/src/main.rs

```diff
@@ -725,10 +717,16 @@ fn run_riscv_asm<F: FieldElement>(
     prove_with: Option<BackendType>,
     export_csv: bool,
     csv_mode: CsvRenderModeCLI,
-    coprocessors: powdr_riscv::CoProcessors,
+    coprocessors: Option<String>,
     just_execute: bool,
     continuations: bool,
 ) -> Result<(), Vec<String>> {
+    let coprocessors = match coprocessors {
+        Some(list) => {
+            powdr_riscv::CoProcessors::try_from(list.split(',').collect::<Vec<_>>()).unwrap()
+        }
+        None => powdr_riscv::CoProcessors::base::<F>(),
```

Comment:
None sounds like there would be no coprocessor. Could we instead use default or something?

Reply:
none just means the user didn't explicitly select any, therefore we use our internal default which is `base`. I think it's better to handle this internally than require the CLI option to know what the default is

Reply:
Not sure what you mean, e.g. in the `RiscvAsm` command we have `coprocessors: Option<String>` because the user doesn't have to specify it. Are you suggesting to make a new enum?